examples/drivenet/drivenet_close_loop.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DriveNet Close Loop Evaluation\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./drivenet_train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "## What is a close loop evaluation?\n",
+    "In close loop evaluation DriveNet is in **full control of the AV**. At each time step, we predict the future trajectory and then we move the AV int he first of the DriveNet's predictions. \n",
+    "\n",
+    "At Lyft, we refer to this process with the term **unroll**\n",
```

Comment:
I think: 'by the term' ... are we keeping unroll? (I think in the paper we have 'forward-simulate')